packages/api/AlvTimeWebApi/Controllers/EconomyData/SalaryController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using AlvTime.Business.EconomyData;
+using AlvTimeWebApi.Authentication;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AlvTimeWebApi.Controllers.EconomyData
+{
+    [Route("api/EconomyData")]
+    [ApiController]
+    public class SalaryController : Controller
```

Comment:
Burde vi hatt en controller for å hente ut utbetalingsinfo også? Altså antall kroner. Kanskje kun gjøre den tilgjengelig for admin foreløpig, men på sikt ser jeg for meg at alle ansatte skal kunne se utbetalingene sine i kroner.

Reply:
Hvilken av utbetalingene tenker du da på? Vanlig lønnslipp eller overtidsutbetalingen? (For hvor mye man får overtidsbetalt er ønsket av flere allerede, tenkte jeg kunne slå meg løs på det etter denne issuen)

Reply:
Oppretter egen issue på dette